Sources/IO/IOStream.swift

```diff
@@ -458,6 +462,7 @@ open class IOStream: NSObject {
     /// A handler that receives stream readyState will update.
     /// - Warning: Please do not call this method yourself.
     open func readyStateWillChange(to readyState: ReadyState) {
+        delegate?.stream(self, willChange: readyState)
```

Comment:
When it's located here, I tend to forget to implement it in the subclass. Therefore, I request the following changes.
```swift
public var readyState: ReadyState = .initialized {
    willSet {
        guard readyState != newValue else {
            return
        }
        delegate?.stream(self, willChange: newValue)
        readyStateWillChange(to: newValue)
    }
    didSet {
        guard readyState != oldValue else {
            return
        }
        readyStateDidChange(to: readyState)
        delegate?.stream(self, didChange: readyState)
    }
}
```